src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/common/PersistenceServiceTest.java

```diff
@@ -1047,4 +1052,11 @@ void postRoleAssignmentsByMultipleQueryRequest() throws IOException {
             .convertEntityToRoleAssignment(page.iterator().next());
 
     }
+
+    @Test
+    void persistActorCacheException() {
+        BatchFailedException exception = mock(BatchFailedException.class);
+        doThrow(exception).when(entityManager).flush();
```

Comment:
I think its just a mock and required to call a business method to check is this type of exception is throwing or not.